inc/signup_notification/namespace.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * Altis CMS Sign-up Blog Notification.
+ *
+ * @package altis/cms
+ */
+
+namespace Altis\CMS\Signup_Notification;
+
+/**
+ * Boostrap setup to remove updates from the admin.
+ */
+function bootstrap() {
+	remove_action( 'after_signup_site', 'wpmu_signup_blog_notification', 10, 7 );
```

Comment:
Is this working ok? I thought this code might run before the default action has been added in WP core.

Reply:
I changed the remove function priority to make sure it runs after it's added. [0eda048](https://github.com/humanmade/altis-cms/pull/249/commits/0eda0489b3aa3cea29a9923ada782b58b0dced80)
For reference, it's [added here](https://github.com/WordPress/WordPress/blob/6496c6174148436cad3211a6392f34953657af89/wp-includes/ms-default-filters.php#L42).

Reply:
Awesome, thanks @joeleenk